### ISSUE TITLE
Ablation experiments for CDN

### ISSUE THREAD

**Issue by hutuo1213** [YueLiao/CDN#27]
Hi, Dr.Liao,

Thank you for your excellent work of your CDN. 
We have a problem where you report the performance of different variants on the test set in the CDN to choose the best framework. For example:
![1](https://user-images.githubusercontent.com/112799241/230781350-bca75146-011a-4c9c-b403-a37060361b1e.png)
Do you divide the validation set for Hico-Det, and if not, can you tell me why you did it? We have encountered unprecedented difficulties.

Thank you very much for your consideration. I look forward to hearing from you soon.

Sincerely yours,
yaoyaosanqi.

**Comment by YueLiao** [comment]
Hi,

This answer can be found in the [Openreview Reviewer LC3p, Q12](https://openreview.net/forum?id=qDrpme0FAi).

**Comment by hutuo1213** [comment]
Sorry I didn't make it clear. I wonder if it is possible to tune hyperparameters based on test set on hico-det?

**Comment by YueLiao** [comment]
We have not tuned hyperparameters based on the test set on hico-det while just showing ablation studies on the HICO-DET test set. However, we think it is somewhat reasonable to tune the hyperparameters based on the HICO-DET test set while its test set plays a validation set role in most HOI detection research papers.

**Comment by hutuo1213** [comment]
Thank you for your reply.

**Comment by hutuo1213** [comment]
We can't tell the essential difference between the two, please let me know. The modules we are proposing are valid, however, I would like to know how many blocks are added consecutively to achieve the best performance. How do I do this?

**Comment by YueLiao** [comment]
The best and most reasonable way is to divide a minimum validation set from the training set to validate the effectiveness of different modules. However, many HOI detection papers directly conduct such experiments in the test set.

**Comment by hutuo1213** [comment]
Thanks a lot
